admin-ui/src/Ingredients/IngredientList.tsx

```diff
@@ -16,16 +16,20 @@ import { Link, useParams } from "react-router-dom";
 
 const IngredientActions = ({ id, mealName }: { id: string; mealName: string }) => {
   return (
-    <TopToolbar style={{ display: 'contents', alignItems: 'center' }}>
-    <Typography variant="body1" style={{marginRight: 'auto'}}> 
-      <span style={{ fontWeight: 'bold' }}>Meal: </span>  
-      <span style={{ fontStyle: 'italic' }}>{mealName}</span>
-    </Typography>
-    <div>
-      <CreateIngredientButton id={id} />
-      <ExportButton />
-    </div>
-  </TopToolbar>
+    <TopToolbar style={{ display: "contents", alignItems: "center" }}>
+      <Typography variant="body1" style={{ marginRight: "auto" }}>
+        <span style={{ fontWeight: "bold" }}>Meal: </span>
+        <span style={{ fontStyle: "italic" }}>
+          <Link to={`/meals`}>
```

Comment:
This is just going to the meals page. We want it to go to the specific meal show page.

Reply:
done, thank you.